documentation/cloud-manager-guide/secure-data-connectors/add-data-connector.md

```diff
@@ -5,25 +5,37 @@ description: Add a data connector
 ---
 # Add a Data Connector
 
-This guide outlines the steps to add a new secure data connector in Environment Operations Center.
+The process to create a new secure data connector and configure it with data source requires the following high-level steps:
```

Comment:
"...and configure it with data source..." seems like a word is missing? Or maybe "source data" instead of "data source"?